doc/source/release-notes/Percona-Server-5.7.25-28.rst

```diff
@@ -0,0 +1,72 @@
+.. rn:: 5.7.25-28
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona Server| |release| on
+|date|. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.25-28/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+	
+This release is based on `MySQL 5.7.25
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-25.html>`_ and includes
+all the bug fixes in it. |Percona Server| |release| is now the current GA
+(Generally Available) release in the 5.7 series.
+	
+All software developed by Percona is open-source and free.
+	
+.. note:: 
+
+   If you're currently using |Percona Server| 5.7, Percona recommends upgrading
+   to this version of 5.7 prior to upgrading to |Percona Server| 8.0.
+
+Bugs Fixed
+================================================================================
+
+- FLUSH commands written to the binary log could cause errors in case
```

Comment:
Must mention the new option, and PS-5405 must be completed too before the release